src/http/handler.ts

```diff
@@ -94,6 +95,18 @@ export const createHttpHandler =
             body: error.message,
           };
         }
+        if (error instanceof Boom.Boom) {
```

Comment:
If you check this by looking for `error.isBoom` you could skip the explicit dependency on Boom. So other people would not make their bundle bigger for no reason.